report.tex

```diff
@@ -64,29 +64,34 @@ \section{Choosing an investment in the JSE}
 
```

Comment:
I think it would be useful to change to Investing in British American Tobacco in the JSE.